go_test/MANIFEST.json

```diff
@@ -515684,10 +516304,128 @@
        }
       ]
      ],
-     "okp_importKey.https.any.js": [
```

Comment:
This change is a result of the manifest update command that happens in this PR automatically

https://github.com/web-platform-tests/wpt-metadata/blob/0f35dcca7b63b04983e9b0207d741184d8a556a3/.github/workflows/update_wpt_manifest.yml#L18-L35